### PR TITLE
Fix svg images not showing on dev

### PR DESCRIPTION
**Pull request by superdav42**
we must set the publicPath is webpack config on dev so css urls are correct.

Also fixes a problem where CC images are not showing.

### CODE REVIEW

File: assets/stylesheets/_fix-image-paths.scss [hunk 1]
```diff
@@ -4,16 +4,19 @@ input[type='checkbox'] {
...
 	}
 }
-$payment-cc-brands: 'amex', 'diners', 'discover', 'jcb', 'mastercard', 'unionpay', 'visa';
-$payment-types: 'alipay', 'bancontact', 'giropay', 'eps', 'ideal', 'paypal', 'p24', 'brazil-tef',
-'wechat', 'sofort';
-@each $brand in $payment-cc-brands {
-	&.is-#{$brand} {
-		background-image: url( '../../wp-calypso/public/images/upgrades/cc-#{$brand}.svg' );
+
+.payment-logo {
```
Comment (harriswong) [inline assets/stylesheets/_fix-image-paths.scss:8]: How come we are adding another class?

Comment (harriswong) [reply assets/stylesheets/_fix-image-paths.scss:8]: I think this fixes the "mastercard/visa card images not showing" problem.

**Comment by harriswong** [comment]
👍 🚢

File: assets/stylesheets/_fix-image-paths.scss (hunk 1 above)
Comment (superdav42) [reply assets/stylesheets/_fix-image-paths.scss:8]: To fix the visa mastercard images. Basically it never worked the way it was.

**Pull request merged**